app/agent/rest/ping.js

```diff
@@ -0,0 +1,31 @@
+angular.module('portainer.agent').factory('AgentPing', [
+  '$resource', 'API_ENDPOINT_ENDPOINTS', 'EndpointProvider', '$q',
+  function AgentPingFactory($resource, API_ENDPOINT_ENDPOINTS, EndpointProvider, $q) {
+    'use strict';
+    return $resource(
+      API_ENDPOINT_ENDPOINTS + '/:endpointId/docker/ping',
```

Comment:
Don't we need to specify the agent version in here as well?

Reply:
following our discussion as we are using this to determine the agent version, we need to use the latest API format (no version), you're right, ignore my comment.